packages/strapi-plugin-content-manager/admin/src/components/SortableList/index.js

```diff
@@ -8,18 +8,29 @@ import Item from './Item';
 
 const SortableList = ({ addItem, buttonData, moveItem, removeItem }) => {
   const { relationsLayout } = useLayoutDnd();
+  const [isDraggingSibling, setIsDraggingSibling] = useState(false);
 
   return (
     <div className="col-4">
-      <SortWrapper>
+      <SortWrapper
+        style={{
+          marginTop: 7,
+          paddingTop: 11,
+          paddingLeft: 5,
+          paddingRight: 5,
+          border: '1px dashed #e3e9f3',
+        }}
+      >
```

Comment:
If SortWrapper is only used here, set the style in the compo. If not, replace it with a div to let the style here.